Resources/Prototypes/_SpaceCats/Electromouse/electromouse.yml

```
@@ -0,0 +1,138 @@
+  - type: MobStateActions
+    actions:
+      Critical:
+      - ActionCritSuccumb
+      - ActionCritFakeDeath
+      - ActionCritLastWords
+  - type: MobThresholds
+    thresholds:
+      0: Alive
+      30: Dead
+  - type: MovementSpeedModifier
+    baseWalkSpeed : 5
```

yamllint (1.35.1)

[warning] too many spaces before colon

(colons)